content/guides/docker-concepts/running-containers/persisting-container-data.md

```diff
@@ -0,0 +1,176 @@
+    $ docker run --name=db -e POSTGRES_PASSWORD=secret -d -v postgres_data:/var/lib/postgresql/data postgres
+    ```
+
+    This will start the database in the background, configure it with a password, and attach a volume to the directory PostgreSQL will persist the database files.
+
+3. Connect to the database by using the following command:
+
+    ```console
+    $ docker exec -ti db psql -U postgres
+    ```
+
+4. In the PostgreSQL command line, run the following to create a database table and insert two records:
+
+    ```psql
+    CREATE TABLE tasks (
+        id SERIAL PRIMARY KEY,
+        description VARCHAR(100)
+    );
+    INSERT INTO tasks (description) VALUES ('Finish work'), ('Have fun');
+    ```
+
+5. Verify the data is in the database by running the following in the PostgreSQL command line:
+
+    ```psql
+    SELECT * FROM tasks;
+    ```
+
+    You should get output that looks like the following:
+
+    ```text
+     id | description
+    ----+-------------
+      1 | Finish work
+      2 | Have fun
+    (2 rows)
+    ```
+
+6. Exit out of the PostgreSQL shell by running the following command:
+
+    ```console
+    \q
+    ```
+
+7. Stop and remove the database container. Remember that, even though the container has been deleted, the data is persisted in the `postgres_data` volume.
+
+    ```console
+    $ docker stop db
+    $ docker rm db
+    ```
+
+8. Start a new container by running the following command, attaching the same volume with the persisted data:
+
+    ```console
+    $ docker run --name=new-db -d -v postgres_data:/var/lib/postgresql/data postgres 
+    ```
+
+    You might have noticed that the `POSTGRES_PASSWORD` environment variable has been omitted. That’s because that variable is only used when bootstrapping a new database.
+
+9. Verify the database still has the records by running the following command:
+
+    ```console
+    $ docker exec -ti new-db psql -U postgres -c “SELECT * FROM tasks”
+    ```
+
+### View volume contents
+
+The Docker Dashboard provides the ability to view the contents of any volume, as well as the ability to export, import, and clone volumes.
+
+1. Open the **Docker Dashboard** and navigate to the **Volumes** view. In this view, you should see the **postgres_data** volume
+
+2. Select the **postgres_data** volume’s name.
+
+3. The **Data** tab shows the contents of the volume and provides the ability to navigate the files. Double-clicking on a file will allow you to see the contents and make changes.
+
+4. Right-click on any file to save it or delete it.
+
+
+### Remove volumes
+
+Before removing a volume, it must not be attached to any containers. If you haven’t removed the previous container, do so with the following command (the `-f` will stop the container first and then remove it):
+
+```console
+$ docker rm -f new-db
+```
+
+There are a few methods to remove volumes, including the following:
+
+1. Select the **Delete Volume** option on a volume in the **Docker Dashboard**.
```

Comment:
Typically ordered lists are for things that must be done in sequence. Consider using a bulleted list for options where only one thing needs to be done.

Reply:
Thanks for the suggestion. I have added the bullet points instead.